lib/simplecov.rb

```diff
@@ -57,9 +57,9 @@ def start(profile = nil, &block)
     # their coverage to zero.
     #
     def add_not_loaded_files(result)
-      if @track_files_glob
+      if track_files
```

Comment:
I think this needs to be `@track_files` (as I don't see an accessor method for track_files defined anywhere).

We have a `track_files nil` configuration in our .simplecov that seems to be skipped after upgrading to the latest version, which led me here.